docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -1,92 +1,123 @@
-
-To add a new user to a deployment:
-
-1. Sign in to your Dagster Cloud account.
+In this guide, we'll cover the available user roles, how permissions are assigned, and how to add and remove users in your organization.
 
-2. Click the **user menu (your icon) > Cloud Settings**.
-
-3. Fill in the following:
-
-   - **Email** - Enter the user's email address
-   - **Role** - Select the [user role](#understanding-user-permissions) for the user. **Note**: With the exception of the **Organization Admin** role, this role will only apply to the full deployment you're adding the user to.
-
-   For example:
-
-   <Image
-   alt="Cloud Settings Interface for Permissions"
-   src="/images/dagster-cloud/user-token-management/cloud_settings_permissions.png"
-   width={842}
-   height={346}
-   />
-
-4. Click **+ Add**.
-
-### Removing a user
+---
 
-To remove a user from a deployment:
+## Understanding user roles
 
-1. Sign in to your Dagster Cloud account.
-2. Click the **user menu (your icon) > Cloud Settings**.
-3. Locate the user in the user list.
-4. Click **Remove**.
-5. When prompted, confirm the removal.
+<TabGroup>
+  <TabItem name="Overview">
 
-**Note**: This won't remove users from other deployments. For example, if a user has been added to both `prod` and `dev` but only removed in `prod`, they'll still be a user in `dev`.
+### User roles
 
----
+Dagster Cloud supports five user roles for several levels of role-based access control:
 
-## Understanding user permissions
+<table>
+  <thead>
+    <tr>
+      <th width="20%; fixed">Role</th>
+      <th width="15%; fixed">Plan</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <strong>Viewer</strong>
+      </td>
+      <td>Enterprise</td>
+      <td>
+        The least permissive role. Grants view access to most things in a
+        deployment, with some exceptions. Refer to the{" "}
+        <strong>User role permissions</strong> tab for more info.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Launcher</strong>
+      </td>
+      <td>Enterprise</td>
+      <td>
+        Includes all Viewer permissions and the ability to launch and terminate
+        runs.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Editor</strong>
+      </td>
+      <td>All plans</td>
+      <td>
+        Includes all Launcher permissions and the ability to:
+        <ul>
+          <li>Manage sensors and schedules</li>
+          <li>Manage environent variables</li>
+          <li>Manage code locations</li>
+          <li>Manage agents and alerts</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Admin</strong>
+      </td>
+      <td>All plans</td>
+      <td>
+        Includes all Editor permissions and the ability to:
+        <ul>
+          <li>Add and edit users</li>
+          <li>Manage user tokens</li>
```

Comment:
i think the conclusion was that editors can manage their own user tokens and admins can manage/revoke others' user tokens (might be encompassed in 'edit users')